tests/tool/__snapshots__/test_get_hr.ambr

```diff
@@ -38,7 +38,7 @@
   '''
   NAMESPACE      NAME            REVISION    CHART                       SOURCE          
   metallb        metallb         4.1.14      metallb-metallb             bitnami         
-  flux-system    weave-gitops    4.0.22      flux-system-weave-gitops    weave-charts    
+  flux-system    weave-gitops    4.0.36      flux-system-weave-gitops    weave-charts    
```

Comment:
Does this match the output of `flux get hr` for a similar use case of showing the namespace of the `HelmRelease`? or should it be the target namespace? Just checking.

Reply:
`flux get hr` returns information on CRDs themselves, so namespace column is populated from `metadata.namespace`, not from `spec.targetNamespace`. So, it's all fine :)